### PR TITLE
Dtd 241- updated all the test following the DTD-241 changes

### PR DESCRIPTION
**Pull request by roccobruno**
Please have a look at https://github.com/hmrc/interest-forecasting/pull/47 if u don't have an idea of the changes made

### CODE REVIEW

File: src/test/resources/features/ifs/GetDebtForTPSSCases.feature [hunk 1]
```diff
@@ -23,7 +23,7 @@ Feature: Debt Calculation For TPSS MainTrans (1525) case (MVP)
     When the debt item is sent to the ifs service
     Then the 1st debt summary will contain
       | interestDueDailyAccrual | interestDueDebtTotal | intRate | unpaidAmountDebt | totalAmountIntDebt | numberOfDays | amountOnIntDueDebt |
-      | 35                      | 284                  | 2.6       | 500000          | 500284            | 8            | 500000             |
+      | 35                      | 249                  | 2.6       | 500000          | 500249            | 8            | 500000             |
```
Comment (rogerball) [inline src/test/resources/features/ifs/GetDebtForTPSSCases.feature:26]: This should be 7 days, I believe?

Comment (rogerball) [inline src/test/resources/features/ifs/GetDebtForTPSSCases.feature]: Should number of days should match num of days in calc window? (7)

**Review by jayasreemca — APPROVED** [review]
LGTM

**Review by beckfordp — APPROVED** [review]
LGTM 👍

**Review by thadeusssewanyana — APPROVED** [review]

**Pull request merged**